### ISSUE TITLE
Mark optional fields (value types) as Nullable

### ISSUE THREAD

**Issue by cluen** [fschick/Keycloak.RestApiClient#7]
Hi,

would be very cool if the optional fields in representation types could be Nullable.

In examle:
`RealmRepresentation` all *Lifespan, *Timeout, *Interval properties. If you create a realm via `realmsAdminApi.PostWithHttpInfoAsync` you have to set those properties to a valid value instead of default value (i.e. Zero for int) otherwise those values are set to Zero in keycloak. If you leave thoses properties `null` the default value in keycloak is applied.

So instead of `public int AccessTokenLifespan` -> `public int? AccessTokenLifespan` would be cool. Maybe it's necessary to remove the `EmitDefaultValue = true` in `DataMember-Attribute`.

Here are some (maybe all) of those properties:
```
                AccessTokenLifespan = 300,
                AccessTokenLifespanForImplicitFlow = 900,
                SsoSessionIdleTimeout = 1800,
                SsoSessionMaxLifespan = 36000,
                OfflineSessionIdleTimeout = 2592000,
                OfflineSessionMaxLifespan = 5184000,
                AccessCodeLifespanUserAction = 300,
                AccessCodeLifespan = 60,
                AccessCodeLifespanLogin = 1800,
                ActionTokenGeneratedByAdminLifespan = 43200,
                ActionTokenGeneratedByUserLifespan = 300,
                Oauth2DeviceCodeLifespan = 600,
                Oauth2DevicePollingInterval = 5,
                MaxFailureWaitSeconds = 900,
                MinimumQuickLoginWaitSeconds = 60,
                WaitIncrementSeconds = 60,
                QuickLoginCheckMilliSeconds = 1000,
                MaxDeltaTimeSeconds = 43200,
                FailureFactor = 30,
                OtpPolicyDigits = 6,
                OtpPolicyLookAheadWindow = 1,
                OtpPolicyPeriod = 30,
```

**Comment by fschick** [comment]
This was a bug #6 and should be solved with 24.0.2. Can you re-check?

**Comment by cluen** [comment]
Thank you and sorry for the inconvenience. Works with 24.0.2
